src/main/java/com/northconcepts/datapipeline/examples/cookbook/WriteToAnExcelFileWithFreezePane.java

```diff
@@ -0,0 +1,33 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import java.io.File;
+
+import com.northconcepts.datapipeline.core.DataEndpoint;
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.excel.ExcelDocument;
+import com.northconcepts.datapipeline.excel.ExcelWriter;
+import com.northconcepts.datapipeline.job.Job;
+
+import org.apache.log4j.Logger;
+
+public class WriteToAnExcelFileWithFreezePane {
```

Comment:
Let's simplify this to just `FreezePanesInExcel`.  

We're more concerned with having the user find what they're looking for versus being explicit that this has to do with writing only.